tests/get-collision-mesh.js

```diff
@@ -0,0 +1,23 @@
+/* global XRPackage, XRPackageEngine */
+const test = require('ava');
+
+const withPageAndStaticServer = require('./utils/_withPageAndStaticServer');
+
+test('get collision mesh of baked wbn - should exist', withPageAndStaticServer, async (t, page) => {
+  const response = await page.evaluate(pageFunction, `${t.context.staticUrl}/assets/baked-xrpk.wbn`);
+  t.true(response);
+});
+
+test('get collision mesh of unbaked wbn - should not exist', withPageAndStaticServer, async (t, page) => {
+  const response = await page.evaluate(pageFunction, `${t.context.staticUrl}/assets/unbaked-xrpk.wbn`);
```

Comment:
If this XRPK is unbaked, why is the test checking the presence of a volume mesh? Shouldn't that be `null`?

Reply:
Oh I see, both cases are tested. Never mind!